about.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" href="./css/style.css">
+    <title>About Us & Our Team</title>
+</head>
+
+<body>
+    <main class="container-fluid bg-gray row mx-0">
```

Comment:
- [ ]  Please kindly add   a menu with links to the various pages, i.e `Home` and `About` page. Please make sure this pages and present and provide links to them in the menu bar as in the design. Please kindly fix that.